exotica_core/include/exotica_core/setup.h

```diff
@@ -90,6 +90,12 @@ class Setup : public Object, Uncopyable
         ret->InstantiateInternal(init);
         return ret;
     }
+    static exotica::ScenePtr CreateScene(const Initializer& init)
+    {
+        exotica::ScenePtr ret = std::make_shared<exotica::Scene>();
+        ret->InstantiateInternal(init);
+        return ret;
+    }
```

Comment:
How is this going to be used? We can't assign a Scene created like this to the Problem. Using the Scene outside of a problem opens it to more problems and misuse. Once a Scene is created inside a problem it can be used and modified directly, no need for creating an independent scene.

Reply:
The `create_scene` / `CreateScene` methods are for parsing and initialising a scene from an initialiser. This is for cases where you want to obtain information about the scene (e.g. number and names of links and joints) before creating the problem. While the scene initialiser allows you to change the loaded scene in some ways, there is no way to obtain information about the scene, other than parsing the URDF yourself.
At the moment it is not meant that this scene is reused in a problem. Hence, there are no side-effects of accessing a scene like this.

Reply:
I agree there are uses where you just want to get Kinematics from a Scene without a problem. I think this requires documentation (to be added) that

(a) A Scene created via this way can only be used for obtaining kinematics etc. - it won't be assignable nor workable with a problem or solver as it's stand-alone.